packages/@ama-sdk/client-fetch/testing/jest.config.ut.js

```diff
@@ -0,0 +1,21 @@
+const path = require('node:path');
+const getJestProjectConfig = require('../../../../jest.config.ut').getJestProjectConfig;
+const rootDir = path.join(__dirname, '..');
+
+/** @type {import('ts-jest/dist/types').JestConfigWithTsJest} */
+module.exports = {
+  ...getJestProjectConfig(rootDir, false),
+  displayName: require('../package.json').name,
+  rootDir,
+  fakeTimers: {
+    enableGlobally: true,
+    // TODO re-enable fake dates when issue fixed https://github.com/sinonjs/fake-timers/issues/437
```

Comment:
Issue seems to be fixed

Reply:
In version 12 but the current version used by jest is 10.
I will change the comment